packages/contracts/slither.ignore.txt

```diff
@@ -0,0 +1,307 @@
+	- ChainStorageContainer.deleteElementsAfterInclusive(uint256,bytes27) (contracts/L1/rollup/ChainStorageContainer.sol)
+getLastSequencerTimestamp() should be declared external:
+	- StateCommitmentChain.getLastSequencerTimestamp() (contracts/L1/rollup/StateCommitmentChain.sol)
+appendStateBatch(bytes32[],uint256) should be declared external:
+	- StateCommitmentChain.appendStateBatch(bytes32[],uint256) (contracts/L1/rollup/StateCommitmentChain.sol)
+deleteStateBatch(Lib_OVMCodec.ChainBatchHeader) should be declared external:
+	- StateCommitmentChain.deleteStateBatch(Lib_OVMCodec.ChainBatchHeader) (contracts/L1/rollup/StateCommitmentChain.sol)
+verifyStateCommitment(bytes32,Lib_OVMCodec.ChainBatchHeader,Lib_OVMCodec.ChainInclusionProof) should be declared external:
+	- StateCommitmentChain.verifyStateCommitment(bytes32,Lib_OVMCodec.ChainBatchHeader,Lib_OVMCodec.ChainInclusionProof) (contracts/L1/rollup/StateCommitmentChain.sol)
+isCollateralized(address) should be declared external:
+	- BondManager.isCollateralized(address) (contracts/L1/verification/BondManager.sol)
+relayMessage(address,address,bytes,uint256) should be declared external:
+	- L2CrossDomainMessenger.relayMessage(address,address,bytes,uint256) (contracts/L2/messaging/L2CrossDomainMessenger.sol)
+setGasPrice(uint256) should be declared external:
+	- OVM_GasPriceOracle.setGasPrice(uint256) (contracts/L2/predeploys/OVM_GasPriceOracle.sol)
+setL1BaseFee(uint256) should be declared external:
+	- OVM_GasPriceOracle.setL1BaseFee(uint256) (contracts/L2/predeploys/OVM_GasPriceOracle.sol)
+setOverhead(uint256) should be declared external:
+	- OVM_GasPriceOracle.setOverhead(uint256) (contracts/L2/predeploys/OVM_GasPriceOracle.sol)
+setScalar(uint256) should be declared external:
+	- OVM_GasPriceOracle.setScalar(uint256) (contracts/L2/predeploys/OVM_GasPriceOracle.sol)
+setDecimals(uint256) should be declared external:
+	- OVM_GasPriceOracle.setDecimals(uint256) (contracts/L2/predeploys/OVM_GasPriceOracle.sol)
+getL1Fee(bytes) should be declared external:
+	- OVM_GasPriceOracle.getL1Fee(bytes) (contracts/L2/predeploys/OVM_GasPriceOracle.sol)
+passMessageToL1(bytes) should be declared external:
+	- OVM_L2ToL1MessagePasser.passMessageToL1(bytes) (contracts/L2/predeploys/OVM_L2ToL1MessagePasser.sol)
+withdraw() should be declared external:
+	- OVM_SequencerFeeVault.withdraw() (contracts/L2/predeploys/OVM_SequencerFeeVault.sol)
+supportsInterface(bytes4) should be declared external:
+	- L2StandardERC20.supportsInterface(bytes4) (contracts/standards/L2StandardERC20.sol)
+mint(address,uint256) should be declared external:
+	- L2StandardERC20.mint(address,uint256) (contracts/standards/L2StandardERC20.sol)
+burn(address,uint256) should be declared external:
+	- L2StandardERC20.burn(address,uint256) (contracts/standards/L2StandardERC20.sol)
+Reference: https://github.com/crytic/slither/wiki/Detector-Documentation#public-function-that-could-be-declared-external[0m
```

Comment:
Not all of these should be ignored some of these should be fixed. 
The issue specified that "This currently reports ~170 issues with the contracts which need to be reviewed and addressed where necessary the others need to be ignored and added to the slither ignore db." 
Also the ones that are ignored, should be done via running slither with triage option rather than via this file. I understood you were trying to get the GH action to triage these however this should be explicitly done by a developer who introduces an issue.